keras/preprocessing/image.py

```diff
@@ -777,6 +787,9 @@ def __init__(self, directory, image_data_generator,
             for subdir in sorted(os.listdir(directory)):
                 if os.path.isdir(os.path.join(directory, subdir)):
                     classes.append(subdir)
+        # User passed in None and no sub-folders, hence FCN. N.B.: Pass in classes=['.'] for improvements.
```

Comment:
I cannot quite understand this comment. Can you clarify?

Also: line too long.